### DISCUSSION TITLE
Views being way to high

### DISCUSSION THREAD

**Discussion by WebKenth** [umami-software/umami#3075] [Q&A]
![image](https://github.com/user-attachments/assets/0e206dc9-669f-4b62-9395-09c568ad9d35)

For context i am doing multiple events per page load
This is done on a shopify site

Every page load shoots a `page_view` event
if its a product page it also shoots a `product_viewed` event
if its a collection page it also shoots a `collection_viewed` event

i can't find any documentation on what constitutes a view, visitor or visits (i may just be blind)

I can guess a Visitor is session based, Visits is how many visits we have across all sessions and views must be the rest? just a total amount? but if i am sending a custom `page_view` event along side the configuration shouldn't the views be equal to the amount of `page_view` events?

In my case i believe there should be even more events than visits, maybe i am missing something? each page load can send 1-5 events, but somehow they trigger multiple views?

**Comment by WebKenth** [comment]
i figured out why views are so high, events are being counted as views on the `Events` page

![image](https://github.com/user-attachments/assets/13dfe9c0-0882-4f4f-9c5c-41945cf06bdc)


![image](https://github.com/user-attachments/assets/07e17841-b5e0-4ef8-8370-6e3ea4ea04d9)

freshly installed, 2 views on overview

go to events page and it shows 5 views, (2 views, 3 events = 5)

**Reply by mikecao** [reply]
This is likely a bug, can you open an issue?
